### ISSUE TITLE
Integrating composer dialogs using Adaptive dialogs

### ISSUE THREAD

**Issue by Keshavnemeli** [microsoft/BotBuilder-Samples#3693]
I'm trying to follow this example from the repository to integrate composer dialogs into my dotnet microsoft bot V4 application.

https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/csharp_dotnetcore/adaptive-dialog/09.integrating-composer-dialogs

When I run the application it's throwing the following exception.

```System.InvalidOperationException: 'InviteParticipants.dialog error: Could not create an instance of type
 Microsoft.Bot.Builder.Dialogs.Dialog. Type is an interface or abstract class and cannot be instantiated. Path 'triggers[0].actions[0].$kind', line 17, position 18.```
 
 Exception
![exception](https://user-images.githubusercontent.com/39478538/152930898-006889d5-4464-4b47-8dcf-4f6e118ccc93.PNG)

Composer Dialog json file. 
![composerDialog](https://user-images.githubusercontent.com/39478538/152930949-b39a83aa-34b6-455b-b40b-2d76e4a653b1.PNG)

Sample root dialog where this exception is thrown. 
https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/adaptive-dialog/09.integrating-composer-dialogs/Dialogs/RootDialog.cs

![sampleRootDialog](https://user-images.githubusercontent.com/39478538/152931282-b576ee30-d0bc-4883-b2fe-f8e9ae3c891e.PNG)

**Comment by dmvtech** [comment]
Hi @Keshavnemeli What version are you using? Version `4.14.1` that is configured in the sample project, or a newer version?

**Comment by dmvtech** [comment]
Hi @Keshavnemeli Can you clarify on the version you are using?

**Comment by Keshavnemeli** [comment]
I'm really sorry for the late reply. I'm attaching a screenshot of all the dependencies of my project[. 
![devbot-dependecies](https://user-images.githubusercontent.com/39478538/154412945-673d2d66-e683-48ce-a0e7-0455b6878a7e.PNG)

**Comment by Keshavnemeli** [comment]
Hi, any update on this?

**Comment by dmvtech** [comment]
Hi @Keshavnemeli My apologies for the delay. 

Is the sample itself failing for you? Or did you make changes (trying to implement in your own bot) and then it failed? I just tried the sample again and it works for me. 

For full clarify; at this point the Bot Framework SDK team is moving away from integration between adaptive/declarative dialogs and classic waterfall/"code first" dialogs. We are suggesting either going with code-first waterfall dialogs (without using adaptive dialogs in code) _or_ using Composer (with adaptive/declarative dialogs). But not mixing the two. The samples that show this type of integration will likely be removed due to this intention.

**Comment by EricDahlvang** [comment]
Hi @Keshavnemeli 

It looks like this project is referencing a variety of Microsoft.Bot.Builder versions:

![image](https://user-images.githubusercontent.com/11055362/158903106-e1e46887-a293-4e6f-8fcb-1ca90fd7cada.png)

Please ensure each of the Microsoft.Bot.Builder.* version is the same.

Also, Microsoft.Bot.Solutions is no longer receiving updates. How is this project utilizing this library?

**Comment by Keshavnemeli** [comment]
@dmvtech No, I'm trying to integrate this in my bot project which is separate from the bot builder sample. Let me downgrade the version for Bot.Builder.Dialogs.Adaptive to 4.14.1 and give it a try.

**Comment by EricDahlvang** [comment]
note: https://www.nuget.org/packages/Microsoft.Bot.Solutions/ has not been updated since 4.13.2 and this library is no longer supported

https://github.com/microsoft/botframework-solutions#the-virtual-assistant-template-and-samples-have-been-replace-by-the-enterprise-assistant-template-in-bot-framework-composer-all-new-projects-should-use-this-template-instead-this-repo-will-be-archived-and-no-longer-actively-supported-on-march-31-2022-the-existing-assets-will-remain-for-reference-but-will-no-longer-be-available-in-the-visual-studio-marketplace-or-nuget-repository

**Comment by Keshavnemeli** [comment]
Hi, @dmvtech Sorry for the late response. The sample still works but I'm getting the same error in my project.  Even if I get it working well there be any support from Microsoft with the Hybrid method (Integrating composer using adaptive dialogs in waterfall)? I'm asking this as the sample mentioned in the post has been removed. If there's no support should I be going forward with this? Does Microsoft want us to use and move to the composer in the future? If so, how would you suggest dealing with migrating the existing bot built on waterfall dialog to the composer?
